src/background/extension.js

```diff
@@ -6,67 +6,30 @@ import { HelpPage } from './help-page';
 import settings from './settings';
 import { SidebarInjector } from './sidebar-injector';
 import { TabState } from './tab-state';
-import { TabStore } from './tab-store';
 
 /**
```

Comment:
The previous comments here described many implementation details and consequently got out of date. The new comments just describe the overall purpose of the class and how to use it.